From dc245b386bb9ad783def75d4962d8cb1370b05b0 Mon Sep 17 00:00:00 2001
From: Michel Hidalgo <michel@ekumenlabs.com>
Date: Wed, 30 Sep 2020 12:19:24 -0300
Subject: [PATCH 2/2] Fix rules.em after bad patch (#2)

---
 debian/rules.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 5b76fe4..18bf2a4 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -60,7 +60,7 @@ override_dh_auto_install: /tmp/rtienv.sh
 	# In case we're installing to a non-standard location, look for a setup.sh
 	# in the install tree and source it.  It will set things like
 	# CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
-	/tmp/rtienv.sh && \
+	. /tmp/rtienv.sh && \
 	if [ -f "@(InstallationPrefix)/setup.sh" ]; then . "@(InstallationPrefix)/setup.sh"; fi && \
 	dh_auto_install
 
-- 
2.25.1

